### DISCUSSION TITLE
Latest version of appimagetool?

### DISCUSSION THREAD

**Discussion by kuiperzone** [AppImage/AppImageKit#1319] [Q&A] [answered]
Just want to confirm that the latest LTS version of appimagetool is still:

https://github.com/AppImage/AppImageKit/releases/tag/13

?

And not, say, this one: https://github.com/AppImage/appimagetool

Thanks

**Comment by probonopd** [comment, marked as answer]
https://github.com/AppImage/appimagetool is the newest. We will no longer publish newer versions of appimagetool to https://github.com/AppImage/AppImageKit at some point.

**Reply by kuiperzone** [reply]
Thank you for replying.

I'd seen this but was unsure as it is marked "Continuous Build Pre-Release".

I'm looking for something that I could treat as a stable "LTS". Is https://github.com/AppImage/appimagetool  OK for this purpose? Are there breaking changes over the AppImageKit release?

**Reply by TheAssassin** [reply]
There are no significant changes. We just imported the code base, basically.

Regarding a release strategy, we'll have to work something out. E.g., date-based releases.

**Reply by kuiperzone** [reply]
Thank you. I'll stick with AppImageKit for the time being, but will keep my eye on appimagetool.

Have always religiously stuck with `major`.`minor`.`patch` versioning. However, am considering the format `year`.`release-num` for future software myself.
